init/pipeline_generic.py

```diff
@@ -1,76 +1,90 @@
+from typing import Any, Dict, Optional
+
 import dlt
-from dlt.sources.helpers import requests
 
+from dlt.sources.helpers.rest_client import paginate
+from dlt.sources.helpers.rest_client.auth import BearerTokenAuth
+from dlt.sources.helpers.rest_client.paginators import HeaderLinkPaginator
+
+# This pipeline demonstrates how to build a simple REST client for interacting with GitHub's API.
```

Comment:
Need to update this as in the pipeline.py file

Reply:
@burnash what do you exactly mean?

Reply:
again: this is generic template. it creates a source with a default parameters and two resources, you can explain that and move github related things into resources